### ISSUE TITLE
Vendure create fails on WSL

### ISSUE THREAD

**Issue by aviadmini** [vendure-ecommerce/vendure#2037]
**Describe the bug**
Vendure create fails on WSL
npx variant worked at first, but now also fails

**To Reproduce**
1. Run `yarn create @vendure store-vendure` on WSL

**Expected behavior**
A starter project is created

**Environment (please complete the following information):**
 - @vendure/core version: 1.9.2
 - Nodejs version: latest 16 and 18 installed by nvm
 - Database: sqlite

**Additional context**
```
Welcome to @vendure/create v1.9.2!

Let's configure a new Vendure project. First a few questions:

✔ Which database are you using? › SQLite
✔ What identifier do you want to use for the superadmin user? … superadmin
✔ What password do you want to use for the superadmin user? … superadmin
✔ Populate with some sample product data? … no / yes

Setting up your new Vendure project in /home/bigboy/aviadmini/monkes/store-vendure
This may take a few minutes...

  ⠹ Installing dependencies
    → Installing @vendure/core, @vendure/email-plugin, @vendure/asset-server-plugin, @vendure/admin-ui-plugin, dotenv, b
…
    Generating app scaffold
  ✖ Installing dependencies
    → An error occurred when installing dependencies.
    Generating app scaffold
    Initializing server
```

**Comment by michaelbromley** [comment]
Hi,
Can you tell me what version of Yarn you have installed please?

**Comment by aviadmini** [comment]
Thank you for a great hint! None of the installation options worked with Yarn (3.4.1) installed, but everything worked again when I downgraded to Yarn 1.22.19 using:
`corepack prepare yarn@1.22.19 --activate`

I got the same console output with the `yarn create @vendure store-vendure --log-level verbose` and latest yarn. Is there a place I can find more logs that I could provide?

P.S. Feel free to close this issue since my problem has been resolved by a yarn downgrade to v1.
